src/nylas-connection.ts

```diff
@@ -300,8 +300,9 @@ export default class NylasConnection {
                   return reject(e);
                 });
             } else if (
-              response.headers.get('content-length') &&
-              Number(response.headers.get('content-length')) == 0
+              (response.headers.get('content-length') &&
+                Number(response.headers.get('content-length')) == 0) ||
+              response.size == 0
```

Comment:
Because as I saw in the `/oauth/revoke` endpoint, having the `content-length` header set is not guaranteed. The response is empty but it returns with no `content-length` header. Since I can't always rely on it being there, I figured I'd fall back on what node-fetch sees in size, and if it's indeed reporting as size=0, then return nothing.